code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -119,7 +121,7 @@
 
 /obj/item/reagent_containers/glass/beaker
 	name = "beaker"
-	desc = "A beaker. Can hold up to 50 units."
+	desc = "A beaker."
```

Comment:
It might be out of scope, but could you add something more to these descriptions? If you don't have any inspiration, you can always ask me or people in the discord!

Reply:
I will have a think, maybe something like, a beaker it holds and allows reagents to react. ?

Reply:
Yeah, or something like it being used in scientific settings or smt!

Reply:
Personally I would go something like "A simple glass beaker." Theres not really anything special to these normal beakers

Reply:
hmmm ok, I will have a think.

Reply:
there, added a bit to the examine text, if too much I will tone it back